### PR TITLE
Update v1beta2 -> v1 and many other minor updates.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: R/PlatinumGenomes-QC/Sample-Level-QC.Rmd [hunk 1]
```diff
@@ -412,14 +413,17 @@ ggplot(ibs) +
...
-* See the `callSetIds` property on the [variants search](https://cloud.google.com/genomics/v1beta2/reference/variants/search) method.
+
+* See the `callSetIds` property on the [variants search](https://cloud.google.com/genomics/reference/rest/v1/variants/search) method.
 
 To entirely remove a genome from a variant set in the Genomics API:
-* See the [callsets delete](https://cloud.google.com/genomics/v1beta2/reference/callsets/delete) method.
-* To delete a callset using a command line tool, see the the `deletecallset` command in [api-client-java](http://github.com/googlegenomics/api-client-java).
+
+* See the [callsets delete](https://cloud.google.com/genomics/reference/rest/v1/callsets/delete) method.
+* To delete a callset using a command line tool, see the the `gcloud alpha genomics callsets` command.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Sample-Level-QC.Rmd]: Missing "delete" from the gcloud command.
"the" is repeated ("see the the")

File: R/PlatinumGenomes-QC/Sample-Level-QC.md [hunk 2]
```diff
@@ -760,14 +761,17 @@ ggplot(ibs) +
...
-* See the `callSetIds` property on the [variants search](https://cloud.google.com/genomics/v1beta2/reference/variants/search) method.
+
+* See the `callSetIds` property on the [variants search](https://cloud.google.com/genomics/reference/rest/v1/variants/search) method.
 
 To entirely remove a genome from a variant set in the Genomics API:
-* See the [callsets delete](https://cloud.google.com/genomics/v1beta2/reference/callsets/delete) method.
-* To delete a callset using a command line tool, see the the `deletecallset` command in [api-client-java](http://github.com/googlegenomics/api-client-java).
+
+* See the [callsets delete](https://cloud.google.com/genomics/reference/rest/v1/callsets/delete) method.
+* To delete a callset using a command line tool, see the the `gcloud alpha genomics callsets` command.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Sample-Level-QC.md]: Missing "delete" from the gcloud command.
"the" is repeated ("see the the")

File: R/PlatinumGenomes-QC/Variant-Level-QC.Rmd [hunk 3]
```diff
@@ -197,13 +197,15 @@ DisplayQueryResults(result)
 # Removing variants from the Cohort
 
 To mark a variant as problematic so that downstream analyses can filter it out:
-* See the [variant update](https://cloud.google.com/genomics/v1beta2/reference/variants/update) method.
+
+* See the [variant patch](https://cloud.google.com/genomics/reference/rest/v1/variants/patch) method.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.Rmd]: Missing "s" in variants: See the [variants patch]

File: R/PlatinumGenomes-QC/Variant-Level-QC.Rmd [hunk 4]
```diff
@@ -197,13 +197,15 @@ DisplayQueryResults(result)
...
 
 To remove variants from BigQuery only:
+
 * Materialize the results of queries that include the non-problematic variants to a new table.
 * Alternatively, write a custom filtering job similar to what we explored in [Part 2: Data Conversion](./Data-Conversion.md) of this codelab.
 
 To entirely remove a variant from a variant set in the Genomics API:
-* See the [variant delete](https://cloud.google.com/genomics/v1beta2/reference/variants/delete) method.
+
+* See the [variant delete](https://cloud.google.com/genomics/reference/rest/v1/variants/delete) method.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.Rmd]: Missing "s" in variants: See the [variants delete]

File: R/PlatinumGenomes-QC/Variant-Level-QC.md [hunk 5]
```diff
@@ -536,13 +534,15 @@ Displaying the first few rows of the dataframe of results:
 # Removing variants from the Cohort
 
 To mark a variant as problematic so that downstream analyses can filter it out:
-* See the [variant update](https://cloud.google.com/genomics/v1beta2/reference/variants/update) method.
+
+* See the [variant patch](https://cloud.google.com/genomics/reference/rest/v1/variants/patch) method.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.md]: Missing "s" in variants: See the [variants patch]

File: R/PlatinumGenomes-QC/Variant-Level-QC.md [hunk 6]
```diff
@@ -536,13 +534,15 @@ Displaying the first few rows of the dataframe of results:
...
 
 To remove variants from BigQuery only:
+
 * Materialize the results of queries that include the non-problematic variants to a new table.
 * Alternatively, write a custom filtering job similar to what we explored in [Part 2: Data Conversion](./Data-Conversion.md) of this codelab.
 
 To entirely remove a variant from a variant set in the Genomics API:
-* See the [variant delete](https://cloud.google.com/genomics/v1beta2/reference/variants/delete) method.
+
+* See the [variant delete](https://cloud.google.com/genomics/reference/rest/v1/variants/delete) method.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.md]: Missing "s" in variants: See the [variants delete]

**Comment by deflaux** [comment]
Pushed fixes to typos.

**Comment by mbookman** [comment]
LGTM

**Pull request merged**